fapolicy_analyzer/glade/main_window.glade

```diff
@@ -249,6 +282,63 @@
                 <property name="position">2</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkStatusbar">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="margin-left">10</property>
+                <property name="margin-right">10</property>
+                <property name="margin-start">10</property>
+                <property name="margin-end">10</property>
+                <property name="margin-top">6</property>
+                <property name="margin-bottom">6</property>
+                <property name="spacing">2</property>
+                <child>
+                  <object class="GtkProgressBar" id="fapaStatusProgressBar">
+                    <property name="visible">False</property>
+                    <property name="can-focus">False</property>
+                    <property name="fraction">0.0</property>
+                    <property name="text" translatable="yes">Percent Complete</property>
+                    <property name="show-text">False</property>
+                  </object>
+                  <packing>
+                    <property name="expand">True</property>
+                    <property name="fill">True</property>
+                    <property name="position">0</property>
+                  </packing>
+                </child>
+                <child>
+                  <object class="GtkLabel">
+                    <property name="visible">True</property>
+                    <property name="can-focus">False</property>
+                    <property name="xpad">4</property>
+                    <property name="label" translatable="yes">fapolicyd:</property>
```

Comment:
I'm not a big fan of the text here, but I don't have a better suggestion at this time.

Maybe instead of showing a label, just make `fapolicyd status` the tooltip on the icon?

Reply:
I'd like to leave it for this spin. Even w/a tool-tip, the user would still need to deviate from their workflow to hover over the image to know what the icon represents. I think once the image icon is converted to an interactive GtkSwitch, the topic should be revisited.